amivapi/tests/users/test_user_cleanup.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Tests for user module.
+
+Test if inactive non-members get deleted from database
+"""
+
+from datetime import datetime, timedelta
+from lib2to3.pgen2 import token
```

Comment:
Unused import should be removed.

Reply:
Ups. VS-Code was apparently faster with adding imports than I was deleting them.